commands/scan/scan.go

```diff
@@ -75,6 +84,11 @@ func (scanCmd *ScanCommand) SetFixableOnly(fixable bool) *ScanCommand {
 	return scanCmd
 }
 
+func (scanCmd *ScanCommand) SetIsJas(isJas bool) *ScanCommand {
```

Comment:
```suggestion
func (scanCmd *ScanCommand) SetRunJasScans(run bool) *ScanCommand {
```
I whould rename for clarity